### ISSUE TITLE
Plotting Speed Issue

### ISSUE THREAD

**Issue by flutech82** [abueide/harry-plotter#90]
Hello,
   first time I tryed to start HP with 6 parallel plots staggered of 30 minutes as I used to do on CHIA BLOCKCHAIN:
my machine run WIN 10 ,  32 gb ram , 1 NVME 1tb , 1 NVME 500 Gb , AMD Ryzen 5
I put temp dir on 1 tb NVME with 4 plots and 2 plots on 500 Gb NVME
and have a very slow plot speed , more than 24 h to plot 4 plots of 1 Tb NVME and 16 hours for 2 plots500 Gb NVME
So I tryed to sart on the same time 1 plot on CHIA BLOCKCHAIN and same on HP on 1tb NVME and same on 500 Gn NVME.
Here attached what's ahappening after 3.5 Hours.
![image](https://user-images.githubusercontent.com/84220972/120889815-3b783c80-c5ff-11eb-9923-3232b36434ec.png)

**Comment by flutech82** [comment]
Upddate after few hours
we can see 1tb NVME is really slow on Harry Plotter worst Chia Blockchain
![image](https://user-images.githubusercontent.com/84220972/120898976-a04a8b80-c62d-11eb-9560-d181a7316f22.png)

**Comment by Unpacking** [comment]
I have noticed this problem plotting on the same Chia + HarryP SSD. At the same time. I don't know if the same thing happens using a single SSD.

**Comment by abueide** [comment]
what versions are you guys running? it sounds like something is getting bottlenecked or windows is making chia use virtual memory. Or maybe you're running one of the older versions which had a bug in the stats screen which caused really really bad cpu usage.

**Comment by flutech82** [comment]
> I have noticed this problem plotting on the same Chia + HarryP SSD. At the same time. I don't know if the same thing happens with only one program at a time.

Pictures I posted was only a test to show performance difference , When I noticed slow issue I was running only HP

**Comment by flutech82** [comment]
> what versions are you guys running? it sounds like something is getting bottlenecked or windows is making chia use virtual memory. Or maybe you're running one of the older versions which had a bug in the stats screen which caused really really bad cpu usage.

I'm running 1.1.14 , I 'd like to plot only with HP , when I noticed slow issue I was running only HP to Plot and Chia to Farm.
I installed newer versions on olders, maybe could I unistall before install newer one??

Thx

**Comment by Unpacking** [comment]
> > what versions are you guys running? it sounds like something is getting bottlenecked or windows is making chia use virtual memory. Or maybe you're running one of the older versions which had a bug in the stats screen which caused really really bad cpu usage.
> 
> I'm running 1.1.14 , I 'd like to plot only with HP , when I noticed slow issue I was running only HP to Plot and Chia to Farm.
> I installed newer versions on olders, maybe could I unistall before install newer one??
> 
> Thx

I always:
1.- Unistall.
2.- Install new.

I don't know if it will be important.

**Comment by abueide** [comment]
Yes try deleting your ~/.harryplotter folder as well as uninstalling and install new version.

I don't know how harry plotter can cause plot times to be slower since it launches the official chia client directly. I suspect there may be another issue in your setup causing these different results.

**Comment by flutech82** [comment]
> Yes try deleting your ~/.harryplotter folder as well as uninstalling and install new version.
> 
> I don't know how harry plotter can cause plot times to be slower since it launches the official chia client directly. I suspect there may be another issue in your setup causing these different results. You could also try comparing lower k values and testing your setup with k25's to try and figure out what the problem might be.

OOOhhh...ok ...sure! If it launch chia client I have to check better....I'll try to unistall and delete Harry plotter installation folder ...then I'll try to launch k25 alone and test...do you have any idea on how fast should be?

thx

**Comment by abueide** [comment]
@flutech82 maybe don't do k25's it probably won't help. actually just try doing a normal plot with harry plotter, then close it while its running and see how long it takes to finish while harry plotter is closed. then do the same thing while harry plotter is opened & compare those times to the official client. this would be faster to test if you use less than k32. I don't know if you'd want to use k25's though since those only take a few minutes and might not give good testing results. So probably try a higher k value.

**Comment by flutech82** [comment]
I tryed to do some test with K25 but Harry Plotter says k32 is minimum !
I put some comment on other issues because I think it is something related to my issue

**Comment by abueide** [comment]
@flutech82 you need to add "--override-k" without quotes to the extra args field.

**Comment by flutech82** [comment]
> @flutech82 you need to add "--override-k" without quotes to the extra args field.

thanks
